src/main/scala/interplay/PlayBuildBase.scala

```diff
@@ -219,22 +219,44 @@ object PlayReleaseBase extends AutoPlugin {
         }
       }
 
-      Seq[ReleaseStep](
-        checkSnapshotDependencies,
-        inquireVersions,
-        runTest,
-        releaseStepTask(playBuildExtraTests in thisProjectRef.value),
-        setReleaseVersion,
-        commitReleaseVersion,
-        tagRelease,
-        publishArtifacts,
-        releaseStepTask(playBuildExtraPublish in thisProjectRef.value),
-        ifDefinedAndTrue(playBuildPromoteBintray, releaseStepTask(bintrayRelease in thisProjectRef.value)),
-        ifDefinedAndTrue(playBuildPromoteSonatype, releaseStepCommand("sonatypeRelease")),
-        setNextVersion,
-        commitNextVersion,
-        pushChanges
-      )
+      if (sbtPlugin.value) {
```

Comment:
I looked at these steps but I'm not sure what I should check for when reviewing. How did you come up with this list?